<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>managewidget</class>
 <widget class="QWidget" name="managewidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>缺货管理</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/new/prefix1/images/main_logo.png</normaloff>:/new/prefix1/images/main_logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#managewidget{background-color: rgb(0, 255, 255);}</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>531</width>
     <height>321</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>商品名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>商品编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>商品数量</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>缺货状态</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string/>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>420</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>421</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>补货商品编号：</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>补货数量：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>补货</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_2">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>刷新</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
